pkg/redirectpolicy/manager_test.go

```diff
@@ -113,9 +113,11 @@ type fakePodResource struct {
 func (fpr *fakePodResource) Observe(ctx context.Context, next func(resource.Event[*slimcorev1.Pod]), complete func(error)) {
 	panic("unimplemented")
 }
+
```

Comment:
Are these white space changes added by a golang style check linter?

Reply:
Yes, these were added by gofumpt (https://github.com/mvdan/gofumpt - stricter gofmt) that can be configured in gopls.

Reply:
I don't think we have it configured in cilium code base, do we? If that's the case, maybe revert them to avoid causing unnecessary conflicts during backports?

Reply:
no we don't have `gofumpt` configured - we're using `gofmt` - even though it is backwards compatible and just stricter.

anyway - i removed the white space changes and some other formatting optimizations  :+1: